mocket/utils.py

```diff
@@ -1,17 +1,26 @@
+from __future__ import annotations
+
 import binascii
 import io
 import os
 import ssl
-from typing import Tuple, Union
+from typing import Tuple, Union, Callable, TYPE_CHECKING, TypeVar, Any, ClassVar
+from typing_extensions import override
```

Comment:
Might want to revert this if we don't want to include the `typing_extensions` dependency as a core dep.